deps/README

```diff
@@ -0,0 +1,109 @@
+This README tracks Pulp dependencies. For each dependency it includes the following:
+
+* Who maintains it
+* What version Pulp requires
+* If the version requires patches, and if so, what each patch performs
+* What platforms the necessary dependency is not available on
+
+
+python-amqp
+-----------
+
+Team owner: bmbouter
+
+kombu 3.0.24 requires amqp>=1.4.6,<2.0, which is not available in EPEL <= 7, or Fedora <= 20.
+
+This requirement may not actually be necessary if amqp becomes available on the necessary platforms [0].
+
+[0]: https://apps.fedoraproject.org/packages/python-amqp/overview/
+
+
+python-billiard
+---------------
+
+Team owner: bmbouter
+
+celery-3.1.11 requires billiard>=3.3.0.17,<3.4, and a new-enough version is not available in
+Fedora <= 20 or EPEL <= 7.
+
+
+python-celery
+-------------
+
+Team owner: bmbouter
+
+Pulp requires celery 3.1, which is not available in Fedora <= 20 or EPEL <= 7.
+
+
+python-kombu
+------------
+
+Team owner: bmbouter
+
+Celery 3.1.11 requires kombu>=3.0.15,<4.0, which is not in Fedora <= 20,
+or EPEL <= 7.
+
+Pulp would like to use the Qpid transport which is available in upstream
+kombu>=3.0.24
+
+
+Permanent Patches:
+
+1212200.patch - This patch will need to continue to be with us forever as
+                it works around a permanent downstream issue whereby
+                ordereddict is installed as part of the Python system
+                library, and not in site-packages. This is not
+                contributed to upstream Kombu.
```

Comment:
I'd like to do something to improve this, but I don't think it' that simple. It's not that the bits are missing it's that [setuptools doesn't know it's there](https://bugzilla.redhat.com/show_bug.cgi?id=1212200#c7)! Since it's already there we can't add bits to solve the problem. Given that, what do you think?